airbyte-integrations/connectors/source-google-analytics-data-api/source_google_analytics_data_api/utils.py

```diff
@@ -71,7 +71,7 @@
 WRONG_METRICS = "The custom report {report_name} entered contains invalid metrics: {fields}. Validate your custom query with the GA 4 Query Explorer (https://ga-dev-tools.google/ga4/query-explorer/)."
 WRONG_PIVOTS = "The custom report {report_name} entered contains invalid pivots: {fields}. Ensure the pivot follow the syntax described in the docs (https://developers.google.com/analytics/devguides/reporting/data/v1/rest/v1beta/Pivot)."
 API_LIMIT_PER_HOUR = "Your API key has reached its limit for the hour. Wait until the quota refreshes in an hour to retry."
-WRONG_CUSTOM_REPORT_CONFIG = "Please check configuration for custom report {report}. {error_response}"
+WRONG_CUSTOM_REPORT_CONFIG = "Please check configuration for custom report {report}."
```

Comment:
d.- get_error_mapping is implemented in GoogleAnalyticsDataApiBaseStream.

Previously, check_connection in SourceGoogleAnalyticsDataApi would format a config error (WRONG_CUSTOM_REPORT_CONFIG) for HTTP errors checking the connection (read records). Now method get_google_analytics_data_api_base_error_mapping handles this task.

Reply:
The internal error_response is added to message when exception is cached